### ISSUE TITLE
make an enum instead of using strings for encrytion

### ISSUE THREAD

**Issue by oroulet** [FreeOpcUa/python-opcua#611]
@kleskjr I was having a short look at your last PR, maybe we should create an enum in uatypes.py instead of using strings in your security_policy variable? It is hard to write these strings without making a mistake...

**Comment by kleskjr** [comment]
I agree that the strings are not an optimal solution. I was planning to construct dictionaries but it didn't look as elegant. What do you have in mind with the enum? A class with variables, e.g., something like:
    None = 0
    Basic128Rsa15_Sign = 1
    ...
